### DISCUSSION TITLE
EKS Hosted runner fails - Updating registration token failed

### DISCUSSION THREAD

**Discussion by maheshevizio** [actions/actions-runner-controller#1636] [Questions]
Not able to get hosted runner working EKS 1.22, helm 3.8.2, certmanager 1.8.2
Keep getting this error "`DEBUG	events	Warning	{"object": {"kind":"Runner","namespace":"self-hosted-runners","name":"runner-deployment-fxh2s-2qx6d","uid":"1e898037-8363-4c82-bad6-91172fd512da","apiVersion":"actions.summerwind.dev/v1alpha1","resourceVersion":"3804626"}, "reason": "FailedUpdateRegistrationToken", "message": "Updating registration token failed"`
[actions-runner-controller-5d66879849-c42zn copy.log](https://github.com/actions-runner-controller/actions-runner-controller/files/9095724/actions-runner-controller-5d66879849-c42zn.copy.log)

**Comment by toast-gear** [comment]
This error has been raised lots of times and it has always been an environmental issue

https://github.com/actions-runner-controller/actions-runner-controller/search?q=Updating+registration+token+failed&type=issues
